en/docs/administer/changing-to-ibm-informix.md

```diff
@@ -122,7 +122,7 @@ Follow the steps below to change the type of the default datasource.
     !!! warning
```

Comment:
This functionality must be available in the 5.9.0 release by default. Check and remove it. Please check and update similar content.